packages/instructors/src/services/cookie.js

```diff
@@ -0,0 +1,9 @@
+export const getCookie = (key) => {
```

Comment:
We should put these services in a 'frontend-services' folder in the project root and import them into students/instructors

Reply:
Creating separate PR for this change.